apps/web/app/[locale]/kanban/page.tsx

```diff
@@ -67,7 +73,10 @@ const Kanban = () => {
 							{t('common.KANBAN')} {t('common.BOARD')}
 						</h1>
 						<div className="flex w-fit items-center space-x-2">
-							<strong className="text-gray-400">08:00 ( UTC +04:30 )</strong>
+							<strong className="text-gray-400">
+								{`(`}
+								{timezone.split('(')[1]}
```

Comment:
@Anishali2 I don't understand what that suppose to show? Can you talk with @KostyaEver to ask him what was idea of that time / timezone etc? What is that value you display and why it's interesting for user to see?